README.md

```diff
@@ -24,11 +28,39 @@ Rules for maintaining our Community Clean and get the most out of it!
 
 ## Things We Don't Tolerate!
 
+1. Hate
 1. Spam.
 2. Duplicated content.
 3. Posts not related to group topic.
```

Comment:
This will number them correctly, it'll save us the hassle later
```suggestion
1. Spam.
1. Duplicated content.
1. Posts not related to group topic.
```